LibuvSharp/UVStream.cs

```diff
@@ -211,7 +212,58 @@ public void Write(ArraySegment<byte> data, Action<Exception> callback)
 			Ensure.Success(r);
 		}
 
-		public void Shutdown(Action<Exception> callback)
+        public void Write(IList<ArraySegment<byte>> buffers, Action<Exception> callback)
```

Comment:
Now that you mention it I also think IEnumerable would be a better fit. I just knew that `System.Net.Sockets.Socket#BeginSend` takes an `IList<ArraySegment<byte>>` and blindly reused that.